src/server.h

```diff
@@ -1745,6 +1745,7 @@ struct redisServer {
     off_t aof_fsync_offset;         /* AOF offset which is already synced to disk. */
```

Comment:
the comment here is misleading.
maybe we'll use this opportunity to change it to 
```suggestion
    off_t aof_fsync_offset;         /* AOF offset which is already requested to be synced to disk. */
```